Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -391,6 +391,12 @@ private static GameObject CreateItem(PxVobItemData vob)
             // It will set some default values for collider and grabbing now.
             // Adding it now is easier than putting it on a prefab and updating it at runtime (as grabbing didn't work this way out-of-the-box).
             var grabComp = vobObj.AddComponent<XRGrabInteractable>();
+
+            if(FeatureFlags.I.dynamicGrab){
```

Comment:
Browser coding.😉
Please reformat code. We use the C# style of brackets on next line.